README.md

```diff
@@ -553,6 +553,7 @@ Table of Contents
   * [Awardspace.com](https://www.awardspace.com) — Free web hosting + a free short domain, PHP, MySQL, App Installer, Email Sending & No Ads.
   * [Freehostia](https://www.freehostia.com) — FreeHostia offers free hosting services incl. an industry-best Control Panel & a 1-click installation of 50+ free apps. Instant setup. No forced ads.
   * [Byet](https://byet.host) — Byet provides you with a massive free, and ad-free load balanced free web hosting service including PHP, MySQL, FTP, Vistapanel & more!..
+  * [versoly.com](https://versoly.com/) — Unlimited Websites, 70+ blocks, 5 templates, custom CSS, favicon, SEO and forms. No custom domain.
```

Comment:
can you please add a short description of what it is? like `SaaS focussed website builder` or whatever? thanks